test/acceptance/tests/fixtures/cases/static-client-inject/patch.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: static-client
+spec:
+  selector:
+    matchLabels:
+      app: static-client
```

Comment:
Do you need these lines? I briefly tested without them and it generated the same?

Reply:
Interesting,  I thought I did but that could have been one of many variations I tried that I don't remember now 😄 Let me try.

Reply:
Yup, didn't need those, thanks!